scripts/deploy.py

```diff
@@ -0,0 +1,73 @@
+"""
+Deployment script for stack
+"""
+
+import sys
+import boto3
+from botocore.exceptions import ClientError
+import json
+
+STACK_NAME = 'drive-datatrust-dev'
+
+def get_secrets(session, secret_id):
+    """
+    Get stack secrets from AWS Secrets Manager
+    :param session: boto3 session
+    :param secret_id: string
+    :return: dict
+    """
+    secretsmanager = session.client('secretsmanager')
+    secrets = json.loads(secretsmanager.get_secret_value(SecretId=secret_id)['SecretString'])
+    formatted_secrets = []
+    for (key, value) in secrets.items():
+        formatted_secrets.append(
+            {"ParameterKey": key, "ParameterValue": value}
```

Comment:
Pascal case is weird for .py. Is that some convention that this environment uses?

Reply:
Yeah, it's an AWS CloudFormation thing. It looks for key/value pairs with the key named `ParameterKey` and the value named `ParameterValue` rather than just looking for key/value pairs. /shrug

Reply:
ugh... i fn hate random caps for no reason. whatevs